python/lsst/fgcmcal/fgcmOutputProducts.py

```diff
@@ -630,17 +431,13 @@ def run(self, dataRefDict, physicalFilterMap, returnCatalogs=True, butler=None):
 
         retStruct = pipeBase.Struct(offsets=offsets,
                                     atmospheres=atmgen)
-        if returnCatalogs:
-            retStruct.photoCalibCatalogs = pcgen
-        else:
-            retStruct.photoCalibs = pcgen
+        retStruct.photoCalibCatalogs = pcgen
 
         return retStruct
 
     def generateTractOutputProducts(self, dataRefDict, tract,
                                     visitCat, zptCat, atmCat, stdCat,
                                     fgcmBuildStarsConfig,
-                                    returnCatalogs=True,
```

Comment:
I see references to `FgcmBuildStarsTask`, which I believe is now gone.  Perhaps do a search...